### PR TITLE
docs(FAQ): add ASGI version of direct upload to cloud

### PR DESCRIPTION
**Pull request by vytas7**
Complete example at https://gist.github.com/vytas7/4ab1317455318db05b309dae75ab1662

### CODE REVIEW

**Comment by vytas7** [comment]
Based on a snippet by @williamkibira on `falconry/user`.

File: docs/user/faq.rst [hunk 1]
```diff
@@ -668,17 +668,37 @@ The `stream` of a body part is a file-like object implementing the ``read()``
 method, making it compatible with ``boto3``\'s
 `upload_fileobj <https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/s3.html#S3.Client.upload_fileobj>`_:
```
Comment (CaselIT) [inline docs/user/faq.rst:669]: we could maybe add also `aioboto3` link? or at least mention that asgi uses that library to wrap the boto3 config

**Review by CaselIT — DISMISSED** [review]
looks ok, left a comment

File: docs/user/faq.rst (hunk 1 above)
Comment (vytas7) [reply docs/user/faq.rst:669]: Right, good idea 👍 I'll add a note.

**Review by CaselIT — APPROVED** [review]
Lgtm

**Pull request merged**